Feature

haraphat01 opened the pull request.

haraphat01 commented:
This is final work for my Newsweek project, built with HTML, CSS, and Boostrap. Kindly review it for your kind approval.

trekab commented:
Hello @haraphat01, it is unfortunate the I have to mark your code review request because your pull request lacks a meaningful name and a short summary.

![Image PR example](https://cdn.filestackcontent.com/osvFJEkQHWQsgDe0PJoQ)

Read more about this from this [guide](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)
_Feel free to slack me for any inquiries_
__Treasure Kabareebe__

haraphat01 commented:
This is my final work on my Bootstrap project. The instruction was to clone the www.newsweek.com, using only HTML, CSS, and Bootstrap, while ignoring any side that uses Javascript or contains sponsored posts, the aim of the project is to develop a fully responsive website that would fit on small, medium, large and, extra-large devices. 
The technology used was basically;

- Html

- CSS and,

- Bootstrap.



The final project is what I'm submitting now to the TSE for review.

trekab commented:
# Changes Required!
Hi @haraphat01, I'd like to start by commending you on a great job done on this project so far and trying your best to keep your repo professional. :+1:  There are some changes I'd like you to look at though. These include;

- [ ] Your navigation bar breaks on small screens. ([clone screenshot](https://raw.githubusercontent.com/trekab/images/master/arafat_akata/clone_sm_header.png), [orignal site screenshot](https://raw.githubusercontent.com/trekab/images/master/arafat_akata/original_sm_nav.png))
- [ ] Top Story should be on top for small screens in the main section. ([clone screenshot](https://raw.githubusercontent.com/trekab/images/master/arafat_akata/clone_sm_header.png), [orignal site screenshot](https://raw.githubusercontent.com/trekab/images/master/arafat_akata/original_sm_nav.png))
- [ ] Your more stories section([screenshot](https://github.com/trekab/images/blob/master/arafat_akata/more_stories_clone.png?raw=true)) breaks on small screen. (compare [original](https://github.com/trekab/images/blob/master/arafat_akata/more_stories_original.png?raw=true))

__Please work on the above changes and submit another code review request__
_Feel free to contact me on slack for any inquiries about this review_ :wink: 
__Treasure Kabareebe__

haraphat01 commented:
This is my final work on my Bootstrap project. The instruction was to clone the www.newsweek.com, using only HTML, CSS, and Bootstrap, while ignoring any side that uses Javascript or contains sponsored posts, the aim of the project is to develop a fully responsive website that would fit on small, medium, large and, extra-large devices.
The technology used was basically;

Html

CSS and,

Bootstrap.

What have I done again?

- I fixed the broken nav-bar
- The Top Story Section is now displaying at the top

- The more stories section has also been fixed

sumancrest0001 commented:
**Nice work @haraphat01 :clap: :clap:**
Breaking points are working correctly but we need to fix a couple of issues related to alignment, styling, and placement of elements to get a green signal from our side. Since it is a clone project, we have to try to achieve a similar design. Typography is essential and let's try to use similar fonts, font-sizes, and font-weights. Kindly make the following changes and submit them for another review. 
**[NOTE] All the requested changes are mandatory.**

sumancrest0001 commented on style.css:
```diff
@@ -0,0 +1,699 @@
+  color: black;
+}
+
+.newsweek a {
+  text-decoration: none;
+  color: #f72210;
+}
+
+.world a {
+  text-decoration: none;
+  color: #f72210;
+}
+
+.world2 a {
+  text-decoration: none;
+  color: #f72210;
+}
+
```
Let's avoid repetition of code and follow the DRY principle. you can do something like this `.world a, .world2 a, .newsweek a {
  text-decoration: none;
  color: #f72210;
}`

sumancrest0001 commented on style.css:
```diff
@@ -0,0 +1,699 @@
+  text-decoration: underline red;
+}
+
+.anchor2 {
+  color: black;
+  font-size: 0.9rem;
+}
+
+.anchor2 a:hover {
+  color: red;
+}
+
+.anchor3 {
+  color: black;
+  font-size: 0.9rem;
+}
```
CSS styling is repeated for anchor2 and anchor3. You can read [this](https://speckyboy.com/good-bad-css-practices/) article as a reference. [Pathright requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png)

sumancrest0001 commented on style.css:
```diff
@@ -0,0 +1,699 @@
+  padding-top: 1em;
+}
+
+.debate-picture1 {
+  width: 50px;
+  height: 50px;
+  position: absolute;
+  right: 0;
+  top: -0.5em;
+}
+
+.debate-picture2 {
+  width: 50px;
+  height: 50px;
+  position: absolute;
+  right: 0;
+  top: 0.5em;
+}
```
Remove all the repeated codes.

sumancrest0001 commented:
Signup section after the latest news is missing in your project. Please have a look 
in your project :point_down: 
![Screenshot from 2020-05-09 11-38-47](https://user-images.githubusercontent.com/37747619/81465386-b1865600-91e9-11ea-8b60-2e66aa8522ea.png)
It should be :point_down: 
![Screenshot from 2020-05-09 11-36-13](https://user-images.githubusercontent.com/37747619/81465392-bc40eb00-91e9-11ea-9d64-a0a0d7f420b7.png)

sumancrest0001 commented:
Try to fix alignment, margin, padding of the newsletter, subscribe, and footer section on a small screen. It looks congested.
original project :point_down: 
![Screenshot from 2020-05-09 11-42-18](https://user-images.githubusercontent.com/37747619/81465492-2f4a6180-91ea-11ea-9fd8-bc46a89f27a9.png)

in your project :point_down: 
![Screenshot from 2020-05-09 11-41-48](https://user-images.githubusercontent.com/37747619/81465497-396c6000-91ea-11ea-8ab7-a4add2d0d9b7.png)

sumancrest0001 commented:
Let's remove the second navbar on a medium screen to make it similar to that of the original one. 
currently in your project :point_down: 
![Screenshot from 2020-05-09 11-46-48](https://user-images.githubusercontent.com/37747619/81465592-d62efd80-91ea-11ea-839b-1bb3465bb730.png)

it should be :point_down: 
![Screenshot from 2020-05-09 11-46-57](https://user-images.githubusercontent.com/37747619/81465600-e1822900-91ea-11ea-9ff8-684002342861.png)

haraphat01 commented:
This is my final work on my Bootstrap project. The instruction was to clone the www.newsweek.com, using only HTML, CSS, and Bootstrap, while ignoring any side that uses Javascript or contains sponsored posts, the aim of the project is to develop a fully responsive website that would fit on small, medium, large and, extra-large devices.
The technology used was basically;

Html

CSS and,

Bootstrap.

What have I done again?

-  Removed Repeated Codes

- Hid the nav bar on medium screen

- Added the sign up section

- added the font family

- Worked on the footer too

I

lekegitrepo commented:
#### @haraphat01 :clap:
#### Good job so far :+1:  you have a few more requirements to fix before this project can be approved :point_down: 

**_1. Fix the horizontal scroll bar on the page._**

**_2. Fix the subscribe section and make it look like the original at each breakpoint. Importantly, the two images of the subscribe content should be positioned like the original_**

**_if you have any doubts or questions contact me on slack._**

### Reviewed by Temitayo Adeleke.

haraphat01 commented:
This is my final work on my Bootstrap project. The instruction was to clone the www.newsweek.com, using only HTML, CSS, and Bootstrap, while ignoring any side that uses Javascript or contains sponsored posts, the aim of the project is to develop a fully responsive website that would fit on small, medium, large and, extra-large devices.
The technology used was basically;

Html

CSS and,

Bootstrap.

What have I done again?

- There's no horizontal overflow on my side, but in case it shows over there, I have hidden it again. 

- I have also added the second picture on the subscription page. As I've discussed with you on slack, the subscribe button change sometimes as the page refreshes and likewise some elements on the original site, I had fixed before, I just changed it to this new one. 

I hope I'd get this project approved this time. Thank you

addod19 commented:
## Pull Request under review by Daniel Larbi Addo (TSE)

haraphat01 commented:
This is my final work on my Bootstrap project. The instruction was to clone the www.newsweek.com, using only HTML, CSS, and Bootstrap, while ignoring any side that uses Javascript or contains sponsored posts, the aim of the project is to develop a fully responsive website that would fit on small, medium, large and, extra-large devices.
The technology used was basically;

Html

CSS and,

Bootstrap.

What have I done again?
![width](https://user-images.githubusercontent.com/48910492/81632879-3bf5d200-9403-11ea-9343-069be87b099b.png)

-  I have changed the width to [100%]

I hope I'd get this project approved this time. Thank you

haraphat01 commented:
This is my final work on my Bootstrap project. The instruction was to clone the www.newsweek.com, using only HTML, CSS, and Bootstrap, while ignoring any side that uses Javascript or contains sponsored posts, the aim of the project is to develop a fully responsive website that would fit on small, medium, large and, extra-large devices.
The technology used was basically;

Html

CSS and,

Bootstrap.

Attached is the screenshot of it looks now on my browser. 


![width2](https://user-images.githubusercontent.com/48910492/81638638-b2013580-9411-11ea-9fb4-4aa3af5dd680.png)

sumancrest0001 commented:
**STATUS: CHANGES REQUIRED
Hello @haraphat01** 

You have done a great job to fix the issues mentioned by the previous TSEs. You are very close to getting a green signal for this project. We have to fix a couple of minor issues. I believe it won't be that difficult considering your expertise.
[Required] Please make sure that the subscriber, newsletter, and footer section looks similar to the original one as we are doing cloning. currently, it looks a little bit off-color. For example, subscribers and newsletter sections should have equal height. Take a look :point_down: 
![Screenshot from 2020-05-12 12-41-45](https://user-images.githubusercontent.com/37747619/81648882-24dcd180-944f-11ea-99dc-45d4ceb6b324.png)
 It should be like  :point_down: 
![Screenshot from 2020-05-12 12-50-59](https://user-images.githubusercontent.com/37747619/81648971-4c339e80-944f-11ea-9cc9-1d12c5bffea8.png)

sumancrest0001 commented:
please fix the alignment of the image and title of the more stories section on each screen size.
currently
![Screenshot from 2020-05-12 12-42-15](https://user-images.githubusercontent.com/37747619/81649095-81d88780-944f-11ea-8bac-816dbb5c9fc4.png)

sumancrest0001 commented:
On the small screen, the image and title of the subscribe section are overlapping. Fix this issue also.
![Screenshot from 2020-05-12 12-43-46](https://user-images.githubusercontent.com/37747619/81649247-bd735180-944f-11ea-9a3b-970165fc929d.png)

haraphat01 commented:
This is my final work on my Bootstrap project. The instruction was to clone the www.newsweek.com, using only HTML, CSS, and Bootstrap, while ignoring any side that uses Javascript or contains sponsored posts, the aim of the project is to develop a fully responsive website that would fit on small, medium, large and, extra-large devices.
The technology used was basically;

Html

CSS and,

Bootstrap.

What have I done again?

- subscriber, newsletter, and footer section fixed

- alignment of the image fixed

- image and title of the subscribe section are overlapping also fixed.

I hope this is approved this time.
